src/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface ThemeContextInterface {
```

Comment:
Let's use `type` instead of `interface`

Reply:
`ThemeContextProps` sinde, I'm already exporting a `ThemeContext` const that is created by `createContext<>()`.

Reply:
The nice thing about typescript it allows you to export a type of a variable with the same name and it is smart enough to figure out wither the imported name (`ThemeContext`) is used as a value or a type.

Reply:
TS could be smart, but we are dumb sometimes. I will keep it explicit.